1.purpose_and_goals.md

```diff
@@ -1,19 +1,19 @@
 
-The central problem this specification seeks to address is how distributed applications can be composed and then successfully handed off to those responsible for operating them. The problem is not so much about how to write programs, but how to take components of a service oriented (or microservice oriented) architecture, and streamline the workflow that surrounds such applications.
+The central problem this specification seeks to address is how distributed applications can be composed and handed off to those responsible for operating them. The problem is not so much about how to write programs, but how to take components of a service oriented (or microservice oriented) architecture, and streamline the workflow that surrounds such applications.
 
 For example, a contemporary cloud application may be composed of dozens of microservices, each responsible for a discrete chunk of what, broadly speaking, is "an application." Such applications need to be configured, deployed, audited, updated, and deleted. Sometimes the application must be treated as a whole, and sometimes finer levels of granularity are required. And most importantly, often such applications are managed not by one person or one team, but by multiple teams who must cooperate to achieve reliability, stability, and timeliness.
 
-This specification provides a description of such applications, where the description is aimed at separating the concerns of developers from the concerns of operators. Furthermore, it suggests patterns and processes for managing such applications. The specification describes a model for cloud native (i.e. highly distributed) cloud applications, encompassing public cloud technologies, on-prem solutions, and IoT/edge technologies. By specifying a common model, this specification provides a foundation for multiple implementations of a shared description of cloud native applications.
+This specification provides a description of such applications, where the description is aimed at separating the concerns of developers from the concerns of operators. Furthermore, it suggests patterns and processes for managing such applications. The specification describes a model for cloud native (i.e. highly distributed) applications, encompassing public cloud technologies, on-prem solutions, and IoT/edge technologies. By specifying a common model, this specification provides a foundation for multiple implementations of a shared description of cloud native applications.
 
 This open specification defines:
 
-- __Component schematics__, where developers declare the operational characteristics of the code they deliver _in infrastructure neutral terms_.
+- __Workload types and definitions__, where platforms declare the schema of a specific type of workload
```

Comment:
How would you feel about using the term "workload kind" instead of "workload type"? They're semantically equivalent, but "kind" fits better with the fact that workloads will now be `kind: Server`, not `type: Server` in configuration documents.

Reply:
https://github.com/oam-dev/spec/pull/304/files#diff-01be24953bc11e69526ef7b88a72f482R136

More context from the draft proposal here.

Reply:
thx,  I tweaked the wording a bit more, please take a look